examples/config-api/config_api.metatags

```diff
@@ -0,0 +1,2 @@
+description: The ballerina/config module in Ballerina provides an API to read configurations from various sources such as files, environment variables and CLI parameters.
+keywords: ballerina, ballerina by example, bbe, config api
```

Comment:
Should we maybe use "configuration" or "config management" instead of "config api"? @pubudu91 wdyt?

Reply:
Maybe add those two words as additional keywords?